docs/1-Using-Fleet/13-Vulnerability-Processing.md

```diff
@@ -0,0 +1,48 @@
+# Vulnerability Processing
+- [Scheduling queries](#scheduling-queries)
```

Comment:
```suggestion
```
I don't think we need this anchor tag :)

Reply:
Whoops, forgot to update these.